### PR TITLE
GRPC-V2 : Get Block Info

### PR DESCRIPTION
**Pull request by tds-1**
## Purpose

Added support for GRPC V2 GetBlockInfo

## Changes

Added support for GRPC V2 GetBlockInfo

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## Unreleased changes
 - Added support for GRPC V2 Get Account Info
 - Added support for GRPC V2 Get Account List
+- Added support for GRPC V2 Get Block Info
```
Comment (EmilLa1) [inline CHANGELOG.md:6]:
```suggestion
- Added support for GRPC V2 'GetBlockInfo' for retrieving information of the block being queried.
```

File: concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java [hunk 2]
```diff
@@ -161,6 +162,18 @@ public ConsensusStatus getConsensusInfo() {
         return to(grpcOutput);
     }
 
+    /**
+     * Retrieves a {@link BlockInfo}
+     *
+     * @param blockHashInput the block {@link BlockHashInput} to query.
+     * @return A {@link BlockInfo} for the block
+     */
```
Comment (EmilLa1) [inline concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java:170]: Please state what will happen if an "unknown" (a block that the node does not know about) block is being queried via the 'GIVEN' constructor.

**Review by EmilLa1 — COMMENTED** [review]
Looks good! GJ.

A few very nitpicky things about phrasing and documentation. But once that is resolved this is good to go.

File: concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java (hunk 2 above)
Comment (tds-1) [reply concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java:170]: Right now it throws StatusRuntimeException, Do we need to change this behaviour? 
Screenshot 2023-04-02 at 7 38 21 PM

**Review by EmilLa1 — APPROVED** [review]
Looks good.

**Pull request merged**